### ISSUE TITLE
clarify workshop orchestration

### ISSUE THREAD

**Issue by tomkralidis** [geopython/geopython-workshop#76]
Draft schedule/timing (time zone is in CEST):

- 8:30: team to meet to ensure no technical issues
- 9:00: start, welcome, workshop team introductions, schedule
- 9:10
  - Section 1. Introduction: @tomkralidis **all**
  - Section 2. Geometry:  @justb4 @krishnaglodha 
  - Section 3. Projections: @ldesousa @justb4 
  - Section 4. Vector data: @justb4 @krishnaglodha 
- 10:30: 10 minute break
- 10:40 
  - Section 5. Raster data: @ldesousa - @francbartoli 
  - Section 6. Data analysis: @francbartoli @krishnaglodha 
  - Section 7. Visualization: @krishnaglodha @justb4 
  - Section 8. Metadata: @tomkralidis @ldesousa
- 12:00: 10 minute break
- 12:10
  - Section 9. Publishing: @tomkralidis @justb4 
  - Section 10. Remote data: @justb4 @francbartoli 
  - Section 11. Emerging technology and trends: @tomkralidis @francbartoli 
  - Section 12. Conclusion: @tomkralidis **all**
- 13:00: close

The above is an initial draft, will make adjustments (for example, sections 1 and 12 will be comparatively shorter).

**Comment by tomkralidis** [comment]
At this point we need folks to volunteer for leading section.  As discussed, we should also have backups identified.

@pvgenuchten @kalxas @justb4 @francbartoli @krishnaglodha @ldesousa I've updated the ticket description with **LEAD - BACKUP**.  Please feel free to update the ticket with sections you are interested in leading.  For reference, last year's delivery is in #67

**Comment by justb4** [comment]
My view:
We are with 6 for 12 sections. Tasks are both authoring and presenting. Plus 'overall tasks' such as Docker-stuff, deployment etc. Easiest is to present what one has authored.

Authoring, well more reviewing/updating current materials,  12/6 would average 2 sections a person (duh).
My understanding:  'Lead' means at least authoring and possibly presenting if one is there. The 'Backup' also carefully follows the authoring, preferably working together, presenting at the event if needed. At least one of Lead+Backup needs to be present.  The entire content would be reviewed by all. 

Hence I subscribed for 2 sections now (I did 7 and 10 as well in last edition) to give others the opportunity (@krishnaglodha Visualisation?) to be lead. I'll be happy to backup e.g. 7 and 10. Thinking about loud here...
